src/dsp_tools/commands/ingest_xmlupload/bulk_ingest_client.py

```diff
@@ -113,3 +113,14 @@ def retrieve_mapping(self) -> str | None:
         print("Ingest process completed.")
         logger.info("Ingest process completed.")
         return res.text
+
+    def finalize(self) -> bool:
+        """Delete the mapping file and the temporary directory where the unprocessed files were stored."""
+        route = f"{self.dsp_ingest_url}/projects/{self.shortcode}/bulk-ingest/finalize"
```

Comment:
Could you state the purpose of this?

Reply:
This is to keep the server tidied up: https://ingest.dev.dasch.swiss/docs/#/bulk-ingest/postProjectsShortcodeBulk-ingestFinalize